Library/Homebrew/formula_auditor.rb

```diff
@@ -836,6 +826,26 @@ def audit_revision_and_version_scheme
       end
     end
 
+    def audit_version_scheme
+      return unless @git
+      return unless formula.tap # skip formula not from core or any taps
+      return unless formula.tap.git? # git log is required
+      return if formula.stable.blank?
+
+      current_version_scheme = formula.version_scheme
+
+      previous_committed, = committed_version_info
```

Comment:
`#committed_version_info` is cached, right?

Reply:
Yup: https://github.com/Homebrew/brew/pull/16466/files#diff-a899f471366b675f5bc416761c9b8820f944db5ca9a5b5fb62a957bbb399d42cR982